lua/iswap.lua

```diff
@@ -11,6 +11,7 @@ local M = {}
 M.config = default_config
 
 function M.setup(config)
+  config = config or nil
```

Comment:
Would it be better to do `config = config or {}`

Reply:
You're absolutely right, my bad